### DISCUSSION TITLE
Archers tweak suggestion

### DISCUSSION THREAD

**Discussion by ChiefDigita1** [ldtteam/minecolonies#9990] [Ideas]
Suggestion for setting up archers. As I understand archers shoot a maximum of 4 blocks, which is why they are practically useless. It would be cool if this could be changed or a config added. I tried to make something like a turret tower, but due to the range it didn’t work

**Comment by Thodor12** [comment]
They are not limited, in fact, they can shoot at most 24 blocks away, that is their effective attack range, the distance they can actually fire ranges somewhere from 10-24 depending on some conditions.
It might look like they never engage from further away because if guards are moving, by the time the AI realizes there's a target in range, they might already be standing next to one another, making it look lik their attack range is bad.

Please do some proper testing beforehand, while trying to lock an archer and a mob in place at distance, and see if they engage, don't allow them to walk.
If they do not engage beyond 4 blocks, then create an issue again, I will close this discussion for now.
